src/v2/components/BlockLightbox/components/BlockLightboxLink/components/BlockLightboxLinkScreenshot/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Box from 'v2/components/UI/Box'
+import Text from 'v2/components/UI/Text'
+import Icons from 'v2/components/UI/Icons'
+
+import { BlockLightboxLinkProps } from 'v2/components/BlockLightbox/components/BlockLightboxLink'
+
+const Screenshot = styled(Box)``
+
+const Image = styled(Box)`
+  position: relative;
+  background-size: cover;
+  background-position: top center;
+  background-image: url(${props => props.backgroundImage});
+
+  &:after {
+    content: '';
+    display: block;
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    height: 6em;
+    background: linear-gradient(
+      ${props => props.theme.colors.utility.transparent} 0%,
```

Comment:
At some point u might want to install the `themeGet` function and just use that which would cut down on the tedium of these things `linear-gradient(${themeGet('colors.utility.transparent')})`